idb/postgres/internal/writer/writer.go

```diff
@@ -220,13 +208,60 @@ func getTransactionParticipants(txn transactions.Transaction) []basics.Address {
 	add(txn.AssetCloseTo)
 	add(txn.FreezeAccount)
 
+	for _, inner := range stxnad.ApplyData.EvalDelta.InnerTxns {
+		getTransactionParticipantsImpl(inner, add)
+	}
+}
+
+// getTransactionParticipants returns referenced addresses from the txn and all inner txns
+func getTransactionParticipants(stxnad transactions.SignedTxnWithAD) []basics.Address {
+	var res []basics.Address
+	var add func(address basics.Address)
+	const acctsPerTxn = 7
+
+	if stxnad.Txn.Type != protocol.ApplicationCallTx {
```

Comment:
or we could test on presence of ApplyData InnerTxns?

Reply:
done